### PR TITLE
Test data for Prod, Test, Dev

### PR DESCRIPTION
**Pull request by dimak1**
- Move sql data imports into its own folders for `Prod`,` Test`, `Dev` environments 
- Update `Makefile` for deploying test data
- Update deploy-data.sh
- Fix imports and tests

### CODE REVIEW

File: schema/data/deploy-data.sh [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  -d, --drop-db
+    Drops the $database database before deploying
+  -s, --deploy-swrs-schema
+    Redeploys the swrs schema and inserts the swrs test reports. This requires the .cas-ggircs submodule to be initialized
+  -p, --deploy-portal-schema
+    Redeploys the portal schema
+  -h, --help
+    Prints this message
+
+EOF
```
Comment (matthieu-foucault) [inline schema/data/deploy-data.sh]: Add the documentation for the new options

File: schema/data/deploy-test-data.sh [hunk 2]
```diff
@@ -156,4 +156,4 @@ _psql -f "./portal/application.sql"
 _psql -f "./portal/certification_url.sql"
 _psql -f "./portal/application_revision.sql"
 _psql -f "./portal/application_revision_status.sql"
-_psql -f "./portal/form_result.sql"
+_psql -f "./portal/form_result.sql"
```
Comment (matthieu-foucault) [inline schema/data/deploy-test-data.sh]: delete this file?

Comment (dimak1) [reply schema/data/deploy-test-data.sh]: I will remove the file itself

File: schema/data/deploy-data.sh (hunk 1 above)
Comment (dimak1) [reply schema/data/deploy-data.sh]: Added documentation for new flags

**Pull request merged**